elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTransaction.java

```diff
@@ -82,11 +82,12 @@ public void createObject(Object entity, RequestScope scope) {
             Query query = buildQuery(entityProjection, scope);
             if (cache != null && !query.isBypassingCache()) {
                 String tableVersion = queryEngine.getTableVersion(query.getTable(), queryEngineTransaction);
-                if (tableVersion != null) {
-                    cacheKey = tableVersion + ';' + QueryKeyExtractor.extractKey(query);
-                    result = cache.get(cacheKey);
-                }
+                tableVersion = tableVersion == null ? "" : tableVersion;
```

Comment:
Will the null case happen when a cache query is not provided?

Would something like this help with debugging?
```suggestion
                tableVersion = tableVersion == null ? "__UNSET_TABLE_VERSION__" : tableVersion;
```

Reply:
null happens when no VersionQuery annotation exists.  The ";" I feel is sufficient to know there was nothing set.